docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -1046,18 +1055,22 @@ code: |
   else:
     waiting_screen_uninstall
   show_interview    
-  button_uninstall_package = True
+  button_uninstall_package = True  
 ---
 id: run package
 event: run_package
 question: |
   Preview your package
 subquestion: |
-
-  If the button below to run the interview does not work, try waiting
-  30 seconds.
-
-  ${ action_button_html(interview_url(i="docassemble." + space_to_underscore(re.sub("\W","",interview.title)) + ":data/questions/" + interview.file_name + '.yml'), label = "Run interview", new_window=True ) }
+  % if not install_step1_status or (defined('task_status') and (not task_status or not task_succeeded)):
+  Something went wrong with package installation, sorry.
```

Comment:
Do we want to show them what went wrong somehow? Or just say to reference the log.